Tests/NIOPosixTests/EventLoopTest+XCTest.swift

```diff
@@ -22,6 +22,7 @@ import XCTest
 /// Do NOT edit this file directly as it will be regenerated automatically when needed.
 ///
 
+@available(macOS 10.15, iOS 13.0, watchOS 6.0, tvOS 13.0, *)
```

Comment:
We can't add these: where did these come from? I think we resolved this in your prior PR.